recipes-core/images/intel-aero-image.bb

```diff
@@ -61,6 +61,18 @@ GRUB_TIMEOUT = "3"
 IMAGE_INSTALL += "librealsense"
 IMAGE_INSTALL += "librealsense-graphical-examples"
 
+#connectivity
+IMAGE_INSTALL += "backport-iwlwifi"
+IMAGE_INSTALL += "linux-firmware-iwlwifi-8000c"
```

Comment:
great... now it's even clear the linux-firmware thing.  On the other PR, check the .dot file that is created... it should tell you who is bringing the entire linux-firmware package